shared/tx-pool-executor/src/tx_pool_executor.rs

```diff
@@ -347,4 +348,54 @@ mod tests {
             )))
         );
     }
+
+    #[test]
+    fn test_verify_and_add_invalid_since_tx_to_pool() {
+        let (shared, always_success_out_point) = setup(10);
+        let last_block = shared
+            .store()
+            .get_block(&shared.lock_chain_state().tip_hash())
+            .unwrap();
+        let last_cellbase = last_block.transactions().first().unwrap();
+        let since_absolute_block_number = |block_number| block_number;
```

Comment:
why?

Reply:
When we want to build a `since` which represents "since an absolute block number":

```
let since = 0b000_0000_0000_0000_0000_0000_0000_0000_0000_0000_0000_0000_0000_0000_0000_0000 | block_number
```

But Clippy warns `0b000...000` is redundant. So I remove `0b000..000` but keep the function.